api/openapi-spec/swagger.json

```diff
@@ -164,15 +164,18 @@
       ],
       "properties": {
         "lastContainerStatuses": {
+          "description": "LastContainerStatuses records the before-in-place-update container statuses. It is a map from ContainerName to InPlaceUpdateContainerStatus",
```

Comment:
Let us be consistent about whether add . in the end of sentence or not in this file.